### DISCUSSION TITLE
Python OMPL module for windows

### DISCUSSION THREAD

**Discussion by Captain-Freels** [community/community#110566] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am having an issue getting the OMPL GUI to work on my computer. When I run 'ompl_app.py' from the software package, I get this error:
File "C:\vcpkg\buildtrees\omplapp\src\1.5.1-6c8b6a14e1.clean\gui\ompl_app.py", line 56, in <module>
    from ompl.util import OutputHandler, useOutputHandler, LogLevel, \
ModuleNotFoundError: No module named 'ompl'

I tried using pip to install it but it keeps saying it doesn't exist. What can I do to mend this?

**Comment by ettaboyle** [comment]
Hi @Captain-Freels,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/zigbee2mqtt). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!
